### PR TITLE
Implement low-hanging-fruit feed card comment frontend tweaks

### PR DESCRIPTION
**Pull request by benhalpern**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

We have had some design improvements sitting unimplemented for some time because they likely require additional functionality adjustments to _fully see them through_.

This is a quick low-hanging-fruit pass at some of the tweaks which do not require full-on new functionality. It does add smarter truncation logic as opposed to using the fade-out we had before, with `See more` inserted either as the truncation of a long paragraph or after other rich content.

Otherwise, it tweaks some display logic to limit the vertical space used up by embeds and/or images.

I also fixed a small style bug on the async-rendered feed cards in general along the way.

## Related Tickets & Documents

- Closes https://github.com/forem/forem/issues/19415

## QA Instructions, Screenshots, Recordings

**Instructions:** If testing locally and wanting to quickly view comments showing up in feed and you don't see any currently visible, you can adjust the `article.rb` min score logic we use in `  has_many :top_comments` to temporarily display extra comments for inspection.

Screen Shot 2023-05-02 at 1 53 36 PM

## Added/updated tests?

- [x] Yes
- [ ] No, and this is why: _please replace this line with details on why tests
      have not been included_
- [ ] I need help with writing tests

## [optional] What gif best describes this PR or how it makes you feel?

apples

### CODE REVIEW

File: app/javascript/articles/components/CommentListItem.jsx [hunk 1]
```diff
@@ -8,11 +8,43 @@ function userProfilePage(username) {
...
+  });
+  return text;
+}
+
 export const CommentListItem = ({ comment }) => (
   <div
     className="crayons-comment cursor-pointer"
     role="presentation"
     onClick={(_event) => {
+      if (_event.target.closest('a')) {
```
Comment (benhalpern) [inline app/javascript/articles/components/CommentListItem.jsx:45]: This a bug in some logic on how clickthroughs on links are handled

File: app/assets/stylesheets/ltags/CommentTag.scss [hunk 2]
```diff
@@ -54,6 +54,9 @@
       text-align: right;
       display: block;
       width: 100px;
+      :not(:first-child) {
```
Comment (benhalpern) [inline app/assets/stylesheets/ltags/CommentTag.scss:57]: General small style bug fix

File: app/javascript/articles/components/CommentsCount.jsx [hunk 3]
```diff
@@ -25,6 +25,7 @@ export const CommentsCount = ({ count, articlePath, articleTitle }) => {
         url={`${articlePath}#comments`}
         icon={commentsSVG}
         tagName="a"
+        className="flex items-center"
```
Comment (benhalpern) [inline app/javascript/articles/components/CommentsCount.jsx:28]: Another small style bug fix

**Comment by anujbhavsar96** [comment]
Looks good, @benhalpern ✨ Just two small suggestions,

1. We can have the same side padding for the comment bubble. Shown with a green block on the left.
2. For inside bubble content, we can have the same side and bottom padding. Shown with a green outline block on the left.

235758107-ffd86c75-35fe-42b2-9b93-4640d877e464

**Review by rt4914 — APPROVED** [review]
LGTM. Thanks @benhalpern

**Review by anujbhavsar96 — APPROVED** [review]
LGTM 🚀

**Review by Ridhwana — APPROVED** [review]
LGTM ✨

**Pull request merged**